Unit Tests/Tests/Redux/Store/ConcreteDependencyTests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+@testable import TestUtilities
+@testable import PusherChatkit
+
+class ConcreteDependencyTests: XCTestCase {
+    
+    func test_initWithInstanceLocator_success() {
+        
+        let dependencies = ConcreteDependencies(instanceLocator: DummyInstanceLocator)
+        
+        XCTAssertNotNil(dependencies.storeBroadcaster as? ConcreteStoreBroadcaster)
+        XCTAssertNotNil(dependencies.store as? ConcreteStore)
+    }
+    
+    func test_initWithInstanceLocatorAndOverride_success() {
```

Comment:
Where do we want to draw a line between tests following BDD style and tests that do not follow such style?

Reply:
I am attempting for all unit tests to follow BDD style.  I haven't given these tests my full attention and I've spotted some cases that aren't tested.  I will address this now.

Reply:
I have now updated this test fixture entirely so I think its clear that all the tests are BDD and the test signatures make clear the "given" context.